hypha/apply/funds/workflow.py

```diff
@@ -1285,7 +1423,7 @@ def phases_matching(phrase, exclude=None):
     },
     "internal-review": {
         "name": _("Internal Review"),
-        "statuses": phases_matching("internal_review"),
+        "statuses": phases_matching("rev_review") + phases_matching("internal_review"),
```

Comment:
Can we not rename `rev_review` to `rev_internal_review` instead?

Reply:
Well, it's not really internal review.  It's external AND internal review.

The problem this solves is that there's a banner on submissions page (maybe gone in future versions) that showed a count of "Internal Reviews".  So we lumped this in there rather than creating a new summary tab that would always show up for other workflow types.

Not really sure what's the best way to do this.

Reply:
Since you want it to `phases_matching` with `*_internal_review` I think it makes sense to name it `rev_internal_review`. This is a machine name and not displayed anywhere in the UI.